cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/.pre-commit-config.yaml

```diff
@@ -0,0 +1,51 @@
+repos:
+    - repo: local
+      hooks:
+          - id: black
```

Comment:
remove or not?

Reply:
I'd keep it for now. Just serves our idea.